File: Web/Models/Employee.cs
namespace Web.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public int Age { get; set; }
        public Gender Gender { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Passport { get; set; }

        public int PositionId { get; set; }
        public Position Position { get; set; }
    }
}

File: Web/ClientApp/src/app/domain/entities/employee.ts
import { Gender, Position } from ".";


export class Employee {
    public id: number;
    public fullName: string;
    public age: number;
    public gender: Gender;
    public address: string;
    public phone: string;
    public passport: string;

    public positionId: number;
    public position: Position;

    public static fromJS(data: any = {}): Employee {
        const entity = new Employee();
        
        entity.id = data.id;
        entity.fullName = data.fullName;
        entity.age = data.age;
        entity.gender = data.gender;
        entity.address = data.address;
        entity.phone = data.phone;
        entity.passport = data.passport;
        entity.positionId = data.positionId;
        entity.position = data.position;

        return entity;
    }
}